index.html

```diff
@@ -1,45 +1,49 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <style>
-      textarea {
-        font-family: monospace;
-        display: block;
-      }
-      textarea:disabled {
-        color: black;
-      }
-    </style>
-    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/codemirror.min.css">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/codemirror.min.css">
   </head>
   <body>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/codemirror.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/mode/javascript/javascript.min.js"></script>
+    <dialog id="loading">
+      Loading...
+    </dialog>
+    <script>
+      loading.showModal();
+      window.addEventListener('error', () => {
+        loading.close();
+      });
+    </script>
+    <!-- github ribbon -->
+    <a id="github" href="https://github.com/engine262/engine262"><img width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1"></a>
     <h1>engine262</h1>
-    <h2>A self-hosted JavaScript engine for development and exploration.</h2>
-    <a href="https://github.com/engine262/engine262">engine262 on GitHub</a>
+    <p>A self-hosted JavaScript engine for development and exploration.</p>
+    <!--<a href="https://github.com/engine262/engine262">engine262 on GitHub</a>-->
 
-    <p>Features</p>
-    <ul id="features"></ul>
+    <main>
+      <section id="input-section">
+        <fieldset><legend>Features</legend>
+          <ul id="featureList"><li><label><input id="autoevaluate" checked type="checkbox">Auto Evaluate</label></li></ul>
```

Comment:
can auto evaluate be separate graphically from the engine features?

I'd probably move auto evaluate, script/module, and the evaluate button to one row.

Reply:
also, auto evaluate isn't clearing the console.

Reply:
sure, but evaluate should probably be its own row since it is not a setting but an action.

Reply:
moved mode and auto evaluate together and fixed clearing output